ui/src/app/view/timestamp-accept-editor/timestamp-accept-editor.component.html

```diff
@@ -0,0 +1,142 @@
+      filterBy="label,value.name" [group]="false">
+      <ng-template let-delayCode pTemplate="selectedItem">
+        <div *ngIf="delayCode.value" class="delayCodeSelectedWrapper">
+          <div class="delayCodeSelected__smdgCode">{{delayCode.value.smdgCode}}</div>
+          <div class="delayCodeSelected__delayType">&nbsp;&middot;&nbsp;{{delayCode.value.delayType}}
+          </div>
+          <div class="delayCodeSelected__delayDetails">&nbsp;&middot;&nbsp;{{delayCode.value.name}}</div>
+          <div class="delayCodeSelected__delayDetails">&nbsp;&middot;&nbsp;{{delayCode.value.description}}</div>
+        </div>
+        <div *ngIf="!delayCode.value">
+          <div>{{delayCode.label}}</div>
+        </div>
+      </ng-template>
+      <ng-template let-delayCode pTemplate="item">
+        <div class="delayCodeDetails" *ngIf="delayCode.value">
+          <div class="delayCodeDetails__smdgCode">{{delayCode.value.smdgCode}}</div>
+          <div class="delayCodeDetails__delayType">{{delayCode.value.delayType}}</div>
+          <div class="delayCodeDetails__delayName">{{delayCode.value.name}}</div>
+          <div class="delayCodeDetails__delaydescription">{{delayCode.value.description}}</div>
+        </div>
+      </ng-template>
+    </p-dropdown>
+  </div>
+
+  <div class="p-field p-col-12 p-md-6"></div>
+
+  <div class="p-field p-col-12 p-md-2"></div>
+  <div class="p-field p-col-12 p-md-5">
+    <span class="p-float-label">
+      <textarea class="commentBox" id="commentbox" pInputTextarea [(ngModel)]="respondingToTimestamp.remark"></textarea>
+      <label for="commentbox">{{ 'general.comment.tooltip' | translate }}</label>
+    </span>
+  </div>
+
+  <div class="p-field p-col-12 p-md-2"></div>
+
+
+  <div class="p-field p-col-12 p-md-2"></div>
+  <!-- Vessel Position -->
+  <div class="p-field p-col-12 p-md-4" [hidden]="!showVesselPosition()">
+    <label for="vesselPositionLatitude">{{ "general.timestamp.vesselPosition.latitude.label" | translate }}:</label>
+    <input id="vesselPositionLatitude" [(ngModel)]="vesselPosition.latitude" pInputText style="width: 100%"
+      type="text" />
+  </div>
+  <div class="p-field p-col-12 p-md-4" [hidden]="!showVesselPosition()">
+    <label for="vesselPositionLongitude">{{ "general.timestamp.vesselPosition.longitude.label" | translate }}:</label>
+    <input id="vesselPositionLongitude" [(ngModel)]="vesselPosition.longitude" pInputText style="width: 100%"
+      type="text" />
+  </div>
+  <div class="p-field p-col-12 p-md-6"></div>
+</div>
+
+<div class="p-grid">
+  <div class="p-col-8">
+  </div>
+  <div class="p-col-4 footer_option" style="justify-content: flex-end">
+    <p-button (onClick)="close()" class="footer_option__buttons" icon="pi pi-times-circle"
+      label="{{'general.close.label' | translate}}" iconPos="left">
+    </p-button>
+    <p-button (onClick)="savePortcallTimestamp(respondingToTimestamp)" class="footer_option__buttons"
```

Comment:
does not exist anymore.

Reply:
If it is `responseTimestamp`, please remove the argument and just `this.responseTimestamp` inside `savePortcallTimestamp`.